### PR TITLE
RSVPs, styling improvements, glob-http

### PR DESCRIPTION
**Pull request by willbach**

### CODE REVIEW

File: make-app.sh [hunk 1]
```diff
@@ -57,12 +58,12 @@ echo "[%zuse 419]" > ${app_name}/sys.kelvin
 
 # Place the desk in given pier.
 echo "deleting old desk"
-rm -r "${path_to_pier}/${app_name}"
+rm -r "${path_to_pier}/${app_name}/*"
```
Comment (taalhavras) [inline make-app.sh:61]: you can't quote the *, I think this should be `"${path_to_pier}/${app_name}"/*` (at least that's what worked for me locally)

File: make-app.sh [hunk 2]
```diff
@@ -57,12 +58,12 @@ echo "[%zuse 419]" > ${app_name}/sys.kelvin
...
 echo "copying over"
-cp -Lr $app_name $path_to_pier
+cp -LR "$app_name/*" "$path_to_pier/$app_name/"
```
Comment (taalhavras) [inline make-app.sh:63]: same here re: quotes and *

**Review by taalhavras — COMMENTED** [review]
Tried this locally with the changes I made to make-app.sh and it didn't work. installing the desk was fine and the output was
```
kiln: installing %ucal locally
gall: installing %timezone-store
gall: installing %ucal-store
ucal-store: attempting to bind /ucal-ics
/ucal-ics bound successfully!
gall: installing %ucal-pull-hook
gall: installing %ucal-push-hook
gall: installing %ucal-sync
docket: fetching %http glob for %ucal desk
> |install our %ucal
```

grid properly displayed the new calendar tile w/the icon (nice!) but trying to open the app led to endless loading. some debug logging in %ucal-store shows
```
[%peek-path-is /x/~undefined/timezone/utc/events/all]
peek bad result
/app/ucal-store/hoon:<[186 23].[186 46]>
/app/ucal-store/hoon:<[186 19].[186 46]>
/app/ucal-store/hoon:<[186 7].[192 50]>
/app/ucal-store/hoon:<[176 5].[193 7]>
/app/ucal-store/hoon:<[175 5].[193 7]>
/app/ucal-store/hoon:<[174 5].[193 7]>
/sys/vane/gall/hoon:<[1.260 26].[1.260 60]>
``` 

so presumably something's not setting a ship somehow on the frontend,

**Comment by willbach** [comment]
ok thanks for those `make-app.sh` comments, I'll update that. 

As for the frontend errors, the issue I think is the `%file-server` code. It interferes with `/session.js` and doesn't set `window.ship` properly on the frontend. I'm not sure how to fix an existing installation.

**Comment by taalhavras** [comment]
we can also probably delete `ucal/sur/file-server.hoon`

**Pull request merged**